patch.sh

```diff
@@ -71,8 +71,9 @@ declare -A object_list=(
     ["415.25"]='libnvcuvid.so'
 )
 
+NVIDIA_SMI="$(which nvidia-smi)"
 
-if ! driver_version=$(/usr/bin/nvidia-smi --query-gpu=driver_version --format=csv,noheader,nounits | head -n 1) ; then
+if ! driver_version=$($NVIDIA_SMI --query-gpu=driver_version --format=csv,noheader,nounits | head -n 1) ; then
```

Comment:
Please add double quotes around `$NVIDIA_SMI` substitution in order to prevent parameter skipping when program is not located.